shared/structures/src/OrganizationMetaData.ts

```diff
@@ -346,6 +347,19 @@ export class OrganizationMetaData extends AutoEncoder {
     })
     registrationPaymentConfiguration = PaymentConfiguration.create({ paymentMethods: [PaymentMethod.PointOfSale] });
 
+    @field({
+        decoder: new MapDecoder(StringDecoder, RecordAnswerDecoder),
+        upgrade: (old: RecordAnswer[]) => {
+            const map = new Map<string, RecordAnswer>();
+            for (const answer of old) {
+                map.set(answer.settings.id, answer);
+            }
+            return map;
+        },
+        ...NextVersion,
+    })
+    recordAnswers: Map<string, RecordAnswer> = new Map();
```

Comment:
Zijn de antwoorden op die vragen alijd publiek? Ik zou het denk ik eerder bij OrganizationPrivateMetaData zetten, anders is de data zichtbaar in de API voor alle bezoekers